### ISSUE TITLE
Dev Server Checker not working in Nuxt Layer

### ISSUE THREAD

**Issue by FlvnMFO** [nuxt/eslint#531]
### Environment

- Operating System: `Darwin`
- Node Version:     `v20.15.0`
- Nuxt Version:     `3.15.2`
- CLI Version:      `3.20.0`
- Nitro Version:    `2.10.4`
- Package Manager:  `npm@10.7.0`
- Builder:          `-`
- User Config:      `devtools`, `modules`, `eslint`
- Runtime Modules:  `@nuxt/eslint@0.7.5`
- Build Modules:    `-`

### Package

@nuxt/eslint

### Reproduction

[Stackblitz](https://stackblitz.com/edit/github-syzxdvwb?file=package.json,nuxt.config.ts,app.vue)

### Describe the bug

When using `@nuxt/eslint` in a **Nuxt Layer** context with the Dev Server Checker enabled, ESLint is not reporting any errors/problems when the dev server is running.

Results when launching the dev server in the minimal reproduction : 
<img width="875" alt="Image" src="https://github.com/user-attachments/assets/4d0a194f-d3f4-4cd6-939d-f01505ddd77e" />

Results when launching the `npm run lint` command in the minimal reproduction : 
<img width="882" alt="Image" src="https://github.com/user-attachments/assets/201d36d8-daa1-4be1-adda-9362884c9a2d" />

I have made a [minimal reproduction](https://stackblitz.com/edit/github-xs5e8bnh?file=package.json,nuxt.config.ts,app.vue) in a Nuxt project to see if the Dev Server Checker is working and no problem in that case.
<img width="868" alt="Image" src="https://github.com/user-attachments/assets/1e6c16d8-f8b0-47ee-b677-846f8d2f09bb" />

### Logs

```shell-script

```

**Comment by andreLuis1506** [comment]
I have the same issue, some help?
